compiler/ballerina-lang/src/main/java/io/ballerina/projects/CompilationOptions.java

```diff
@@ -67,6 +68,7 @@ boolean sticky() {
         return toBooleanTrueIfNull(this.sticky);
     }
 
+    @Deprecated
```

Comment:
We dont need to deprecate package private methods

Reply:
Added

Reply:
This seems to still exist. We can try to remove this option since it isnt public

Reply:
Removed